.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,41 @@
+<component name="ProjectCodeStyleConfiguration">
+  <code_scheme name="Project" version="173">
```

Comment:
We should consider adding `<option name="LINE_SEPARATOR" value="&#10;" />` -> \n as separator

Reply:
Added.